search/Sta.cc

```diff
@@ -2069,9 +2069,9 @@ Sta::checkExceptionToPins(ExceptionTo *to,
 			  int line) const
 {
   if (to) {
-    PinSet::Iterator pin_iter(to->pins());
-    while (pin_iter.hasNext()) {
-      const Pin *pin = pin_iter.next();
+    PinSet::Iterator (to->pins());
+    while (.hasNext()) {
```

Comment:
This change is gratuitous and uses some weird syntax I am not familiar with or really approve of.

Reply:
Oh that must've been my linter that added that syntax, it wasn't intended I will revert it